editor/editor_interface.cpp

```diff
@@ -238,6 +238,10 @@ void EditorInterface::popup_dialog_centered_clamped(Window *p_dialog, const Size
 	p_dialog->popup_exclusive_centered_clamped(EditorNode::get_singleton(), p_size, p_fallback_ratio);
 }
 
+void EditorInterface::set_current_feature_profile(const String &p_profile_name) {
+	EditorNode::get_singleton()->set_current_feature_profile(p_profile_name);
```

Comment:
`EditorFeatureProfileManager` is a singleton, we don't need to go through `EditorNode`.